### DISCUSSION TITLE
Image preview when link is copied

### DISCUSSION THREAD

**Discussion by RithwikU** [alshedivat/al-folio#1778] [Q&A] [answered]
```
serve_og_meta: false# Include Open Graph meta tags in the HTML head
serve_schema_org: false # Include Schema.org in the HTML head
og_image: # The site-wide (default for all links) Open Graph preview image
```
Should I make changes to these flags to enable the image preview when I copy my website link?

**Comment by george-gca** [comment, marked as answer]
Absolutely. By setting `serve_og_meta` to true and setting `og_image` you achieve that. For more information about what these do, take a look [here](https://ogp.me/) and [here](https://developers.google.com/search/docs/appearance/structured-data/intro-structured-data).

**Reply by RithwikU** [reply]
Thanks a lot!

**Comment by kallenosf** [comment]
Is there a way to change to have a different preview image for each blog post link?  For example, the thumbnail photo  of the post.

**Reply by george-gca** [reply]
Currently we look for a `og_image` in the page/post front matter, as can be seen here:

https://github.com/alshedivat/al-folio/blob/c5d484ec9faa6adb070e3162b5ff41b98163794b/_includes/metadata.liquid#L61-L63
